kbase-extension/static/kbase/js/widgets/narrative_core/upload/importSetup.js

```diff
@@ -219,9 +227,9 @@ define([
          *   }
          * }
          */
-        const typeToSpecParams = Object.entries(appIdToType).reduce(
-            (typeToSpecParams, [appId, dataType]) => {
-                const spec = appIdToSpec[appId];
+        const typeToAlteredParams = Object.entries(appIdToType).reduce(
```

Comment:
sonar cloud says that this is already defined in the upper scope

Reply:
Oh, I see how I named things. Fixed!